src/msw/mediactrl_am.cpp

```diff
@@ -1530,7 +1530,7 @@ void wxAMMediaBackend::Move(int WXUNUSED(x), int WXUNUSED(y),
 //---------------------------------------------------------------------------
 void wxAMMediaEvtHandler::OnActiveX(wxActiveXEvent& event)
 {
-    switch(event.GetDispatchId())
+    switch ((unsigned long)event.GetDispatchId())
```

Comment:
Sorry for nitpicking, but why not use more greppable `static_cast<>` instead? I'd also add a comment explaining why do we do this.

Reply:
I did try `wxStaticCast`, but when that failed I just used c-style cast. I'll update it.

Reply:
FWIW `wxStaticCast` is to `static_cast<>` what `wxDynamicCast` to `dynamic_cast<>`, i.e. confusingly named similar but not at all identical macro.